src/toast/ops/demodulation.py

```diff
@@ -129,6 +129,11 @@ class Demodulate(Operator):
         "hamming", help="Window function name recognized by scipy.signal.firwin"
     )
 
+    keep_dets_frac = Float(
```

Comment:
I don't see the rationale of this being a part of demodulation but I do appreciate the functionality.